Testnet/kichain-t-4/README.md

```diff
@@ -83,45 +84,48 @@ cd $NODE_ROOT
 
 Initiate the needed configuration files using the `unsafe-reset-all` command:
 ```bash
-kid unsafe-reset-all --home ./kid
+kid tendermint unsafe-reset-all
 ```
 
-Curl the genesis file to the `./kid/config/` folder:
+Curl the genesis file to the `~/.kid/config/` folder:
 ```bash
-curl https://raw.githubusercontent.com/KiFoundation/ki-networks/v0.1/Testnet/kichain-t-4//genesis.json > ./kid/config/genesis.json
+curl https://raw.githubusercontent.com/KiFoundation/ki-networks/v0.1/Testnet/kichain-t-4/genesis.json > ~/.kid/config/genesis.json
 ```
 
 The hash of the genesis file can be computed as follows:
 ```bash
-jq -S -c -M '' ./kid/config/genesis.json | shasum -a 256
+jq -S -c -M '' ~/.kid/config/genesis.json | shasum -a 256
 
 83e8d8b9b883f1d478fb8d9f0e32c138904d0cbc30e939fa0490f988fd0b78d7
 ```
 
-In the file `config.toml` that can be found in the `./kid/config/` directory, make sure to add the provided seed and the community peers found in [`seed.txt`](https://github.com/KiFoundation/ki-networks/blob/v0.1/Testnet/kichain-t-4//seeds.txt) and [`peers.txt`](https://github.com/KiFoundation/ki-networks/blob/v0.1/Testnet/kichain-t-4//peers.txt) by filling the `seeds` and `persistent_peers` fields resp.
+In the file `config.toml` that can be found in the `~/.kid/config/` directory, make sure to add the provided seed and the community peers found in [`seed.txt`](https://github.com/KiFoundation/ki-networks/blob/v0.1/Testnet/kichain-t-4//seeds.txt) and [`peers.txt`](https://github.com/KiFoundation/ki-networks/blob/v0.1/Testnet/kichain-t-4//peers.txt) by filling the `seeds` and `persistent_peers` fields resp.
 
-Make sure to set your `min-gas-price` to `0.025utki` in `app.toml`.
+Make sure to set your `minimum-gas-prices` to `0.025utki` in `app.toml`.
 
 Your node is now ready to be started. If you've started the node with a service you can simply start the service. Otherwise, use the following command:
 
 ```bash
-kid start --home ./kid/ &> ./kilogs/ki-node.log &
+kid start --home ~/.kid/ &> ~/.kilogs/ki-node.log &
 ```
 
 This command will start the block synchronization process where your node retrieves the current state of the blockchain from the other nodes. The process output is redirected to the `ki-node.log` log file. It can be visualized as follows:
 
 ```bash
-tail -f kilogs/ki-node.log
+tail -f ~/.kilogs/ki-node.log
 ```
 
+In case of error, you might need to sync the data. To do so, either ask for a fellow validator's data+wasm folders, or build the v2, sync the blockchain with it (until the upgrade block), then shutdown the v2 and start the v3 in place.
```

Comment:
You might want to put this before the `kid start` command, so people do not try to start before being aware of data retrieval concerns.

```suggestion
To get started, you might need to sync the data. To do so, you can ask a fellow validator for a snapshot containing data+wasm folders. 
You can also sync from the chain start. For this you will need to build and run kid `2.0.0-testnet` version, up until the block `4351400`, and then switch to the `3.0.0-beta` version.
More information on the v3 upgrade [here](https://github.com/KiFoundation/ki-networks/blob/v0.1/Testnet/kichain-t-4/UPGRADE_V3.md)
```

Reply:
Is it better, now ?